Jim-Hodapp-Coaching/refactor-platform-fe
# Add a dialog box to select current and default organization and coaching relationship

## Description
This PR adds a dialog box allowing the current user to select the current and default organization and coaching relationship to (coming soon) apply to the user's session.

#### GitHub Issue: Closes #14 

### Changes
* Adds a dialogue box concept to select the user's current Organization and Coaching Relationship
* Allows a user to update their default Organization and Coaching Relationship to use when they next log in

### Screenshots

<img width="652" alt="Screenshot 2024-03-24 at 22 03 06" src="https://github.com/Jim-Hodapp-Coaching/refactor-platform-fe/assets/3219120/c5225b82-8660-48d2-b258-6092ce6e4090">

### Testing Strategy
1. Log in
2. From the coaching session page, hover your mouse over **Refactor Coaching** menu in the upper left hand corner
3. Select "Select Organization" from the menu
4. Observe the dialogue box appears.


### Concerns
None
